librz/main/rizin.c

```diff
@@ -1362,17 +1352,7 @@ RZ_API int rz_main_rizin(int argc, const char **argv) {
 		rz_config_set (r->config, "scr.prompt", "false");
 	}
 	r->num->value = 0;
-	if (patchfile) {
-		char *data = rz_file_slurp (patchfile, NULL);
-		if (data) {
-			ret = rz_core_patch (r, data);
-			rz_core_seek (r, 0, true);
-			free (data);
-		} else {
-			eprintf ("[p] Cannot open '%s'\n", patchfile);
-		}
-	}
-	if ((patchfile && !quiet) || !patchfile) {
+	if (!quiet) {
```

Comment:
mmm i think this should be always done.
The condition becomes:
`if ((false && !quiet) || !false)` which is equal to `if (false || true)` -> `if (true)`. Right?